README.md

```diff
@@ -1,9 +1,21 @@
 <img src="https://www.kubeflow.org/images/logo.svg" width="100">
-Kubeflow is a Cloud Native platform for machine learning based on Google’s internal machine learning pipelines.
+Kubeflow the cloud-native platform for machine learning operations - pipelines, training and deployment.
 
 ---
+
+## Documentation
 Please refer to the official docs at [kubeflow.org](http://kubeflow.org).
 
+## Working Groups
+The Kubeflow community is organized into working groups (WGs) with associated repositories, that focus on specific pieces of the ML platform. 
+
+* [AutoML](https://github.com/kubeflow/community/tree/master/wg-automl)
+* [Deployment](https://github.com/kubeflow/community/tree/master/wgdeployment)
```

Comment:
This link is missing a dash, it should be `https://github.com/kubeflow/community/tree/master/wg-deployment`

Reply:
@RFMVasconcelos  please fix this before merging 
/hold

Reply:
Fixed!